admin-dev/themes/new-theme/js/pages/product/product-map.ts

```diff
@@ -102,17 +103,17 @@ export default {
     tableRow: {
       combinationImg: '.combination-image',
       deltaQuantityWrapper: '.delta-quantity',
-      combinationCheckbox: (rowIndex) => `${combinationListId}_combinations_${rowIndex}_is_selected`,
-      combinationIdInput: (rowIndex) => `${combinationListId}_combinations_${rowIndex}_combination_id`,
-      combinationNameInput: (rowIndex) => `${combinationListId}_combinations_${rowIndex}_name`,
-      referenceInput: (rowIndex) => `${combinationListId}_combinations_${rowIndex}_reference_value`,
-      impactOnPriceInput: (rowIndex) => `${combinationListId}_combinations_${rowIndex}_impact_on_price_value`,
-      finalPriceTeInput: (rowIndex) => `${combinationListId}_combinations_${rowIndex}_final_price_te`,
-      quantityInput: (rowIndex) => `${combinationListId}_combinations_${rowIndex}_delta_quantity_quantity`,
-      deltaQuantityInput: (rowIndex) => `${combinationListId}_combinations_${rowIndex}_delta_quantity_delta`,
-      isDefaultInput: (rowIndex) => `${combinationListId}_combinations_${rowIndex}_is_default`,
-      editButton: (rowIndex) => `${combinationListId}_combinations_${rowIndex}_edit`,
-      deleteButton: (rowIndex) => `${combinationListId}_combinations_${rowIndex}_delete`,
+      deltaQuantityInput: (rowIndex: number): string => `${combinationListId}_combinations_${rowIndex}_delta_quantity_delta`,
```

Comment:
Just out of curiosity, why change the order? They are not ordered alphabetically so why?
I don't care about the order, it's just to know it there is a pattern I'm missing here 😛

Reply:
Probably just because of the rebase